f5_heat/resources/common/mixins.py

```diff
@@ -14,6 +14,21 @@
 #
 
 
+def f5_common_resources(func):
+    def func_wrapper(self, *args, **kwargs):
+        self.get_bigip()
+        self.set_partition_name()
+        return func(self, *args, **kwargs)
+    return func_wrapper
+
+
+def f5_bigip(func):
+    def func_wrapper(self, *args, **kwargs):
+        self.get_bigip()
+        func(self, *args, **kwargs)
+    return func_wrapper
+
+
```

Comment:
I infer by the presence of the "self" param that these are "method_wrapper" 's not "func_wrapper" 's.